tests/Agent/IntegrationTests/Applications/BasicMvcApplication/BasicMvcApplication.csproj

```diff
@@ -122,7 +108,7 @@
       <Version>5.2.3</Version>
     </PackageReference>
     <PackageReference Include="Microsoft.AspNet.Webpages">
-      <Version>3.2.0</Version>
+      <Version>3.2.7</Version>
```

Comment:
Was there a reason we chose to update the versions here?

Reply:
One of the warnings was about a package version downgrade; this change eliminated the warning and the tests still build and pass.